CHANGELOG.rst

```diff
@@ -25,6 +25,9 @@ Bug Fixes
   _fixtureinfo attribute (e.g. doctests) (`#2788
   <https://github.com/pytest-dev/pytest/issues/2788>`_)
 
+- Strip whitespace from the markers in INI config (`#2856
```

Comment:
We use `towncrier` for changelog management to avoid conflicts, so instead of this please create a file `changelog/2856.bugfix` with this text:

```
Strip whitespace from marker names when reading them from INI config.
```

Reply:
Oh, sorry! I should have paid more attention.